qchem/pennylane_qchem/qchem.py

```diff
@@ -561,6 +561,87 @@ def generate_hamiltonian(
     return convert_hamiltonian(h_of), nr_qubits
 
 
+def sd_excitations(n_electrons, n_orbitals, delta_sz):
+    r"""Generates single and double excitations from a Hartree-Fock (HF) reference state.
+
+    The singly- and doubly-excited configurations are generated by acting with the operators
+    :math:`\hat T_1` and :math:`\hat T_2` on the HF state:
+
+    .. math: 
+        && \vert \Phi_\mathrm{S} \rangle = \hat{T}_1 \vert \mathrm{HF} \rangle = \sum_{r \in
+        \mathrm{occ} \\ p \in \mathrm{virt}} \hat{c}_p^\dagger \hat{c}_r \vert \mathrm{HF} 
+        \rangle \\ 
+        && \vert \Phi_\mathrm{D} \rangle = \hat{T}_2 \vert \mathrm{HF} \rangle = \sum_{r>s \in 
+        \mathrm{occ} \\ p>q \in \mathrm{virt}} \hat{c}_p^\dagger \hat{c}_q^\dagger 
+        \hat{c}_r \hat{c}_s \vert \mathrm{HF} \rangle
+
+	where the indices :math:`r, s` and :math:`p, q` run over the occupied (occ) and unoccupied, 
+	referred to as virtual (virt), molecular orbitals and :math:`\hat c` and 
+	:math:`\hat c^\dagger` are the electron annihilation and creation operators, respectively. 
+
+    **Example**
+
+    >>> ph, pphh = sd_configs(2, 4, 0)
+    >>> print(ph)
+    [[0, 2], [1, 3]]
+    >>> print(pphh)
+    [[0, 1, 2, 3]]
+
+    Args: 
+        n_electrons (int): number of active electrons 
+        n_orbitals (int): number of active orbitals
+        delta_sz (int): Spin-projection selection rule. 
```

Comment:
Is the choice of the selection rule always straightforward? Is it possible to have a default value for it?

Reply:
Yes, and I think it is convenient. The natural default is to stay in the spin sector of the HF reference state, that is, ``delta_sz=0``. 
I had it before but as we decided to code up UCCSD as template, it was gone.
Thanks @soranjh .